Lesson_20/page-objects/basket.js

```diff
@@ -0,0 +1,68 @@
+const { Base } = require('./base');
+
+class Basket extends Base {
+  constructor(page) {
+    super(page);
+  }
+
+  get basketUrl() {
+    return 'https://oz.by/checkout/#';
+  }
+
+  get basketButton() {
+    return this.page.locator('.link.user-bar__item.user-bar__cart');
+  }
+
+  get itemsInBasket() {
+    return this.page.locator('div.goods-table-cell__link');
+  }
+
+  get selectAllButton() {
+    return this.page.locator('span.i-checkbox__faux:last-child');
+  }
+
+  get deleteButton() {
+    return this.page.locator('.i-button.i-button_danger.i-button_small.remove');
+  }
+  get confirmDeletingButton() {
+    return this.page.locator('.i-button.i-button_danger.i-button_small.remove-yes');
+  }
+
+  get emptyBasketMessage() {
```

Comment:
async getItemSelector(numberOfItems) {
    return this.page.locator(`.products__item.item.product-card:nth-of-type(${numberOfItem})>div.product-card__footer>[data-controller="add-to-cart"]>[type="submit"]`)
}